src/ast/mod.rs

```diff
@@ -4706,6 +4706,9 @@ impl fmt::Display for CloseCursor {
 #[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
 pub struct Function {
     pub name: ObjectName,
+    /// The parameters to the function, including any options specified within the
+    /// delimiting parentheses.
```

Comment:
Can we include a link to the clickhouse docs?

Reply:
@iffyio Done